.circleci/config.yml

```diff
@@ -122,6 +123,7 @@ jobs:
           name: Build example APKs
           command: |
             melos run build:examples:android
+          no_output_timeout: 20m
```

Comment:
Nice. Just out of curiousity - any particular reason we arrived at this number?

Reply:
No particular reason. Some of the android tasks are taking upwards of 10 mins which kills the job and doesn't even tell us which one. This will at least complete the job and in the logs we can tell which ones are taking longer to troubleshoot them separately.